RELEASE-NOTES.md

```diff
@@ -1,5 +1,12 @@
 # Wikibase DataModel Serialization release notes
 
+## 3.0.0 (2017-02-16)
+
+* Changed the `DeserializerFactory` constructor to expect a list of Entity deserializers
+* Changed the `SerializerFactory` constructor to expect a list of Entity serializers
+* `DeserializerFactory::newEntityDeserializer` does not return default Item and Property deserializers any more
+* `SerializerFactory::newEntitySerializer` does not return default Item and Property serializers any more
```

Comment:
#212 and the release notes here should have been one pull request. I revoked both.

The problem I'm describing above is still relevant, but can not be solved in this component. I believe it must be solved by *not* using the `newEntity…` methods, but constructing custom dispatching (de)serializers instead. I believe we will introduce a third top-level factory (in addition to the client and repo ones) for this that is aware of custom entity types.

Input on this subject is very much appreciated.